src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,47 @@
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__list {
+    display: flex;
+  }
+
+  &__items {
+    display: flex;
+    position: relative;
+    align-items: center;
+  }
+
+  &__items:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    line-height: 60px;
+    font-weight: $ff-500;
+    font-size: 12px;
+    text-transform: uppercase;
+    color: $description-color-text;
+
+    transition: color 0.7s cubic-bezier(0.52, 0.17, 0, 1.31);
```

Comment:
But you already have variable with duration, better to make your transitions consistent with other transition on the page (it will be strange if hover effect for some element will be shown faster than for other)